packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -52,10 +52,11 @@
           </oc-button>
         </div>
         <file-info
-          v-if="isSingleResource"
+          v-if="isSingleResource && !highlightedFileIsSpace"
```

Comment:
Check should not be necessary if you fix the `isSingleResource` prop (see comment there).